server/routes/imageUpload.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const { validate } = require("../middlewares/validate");
+const router = express.Router();
+const upload = require("../services/imageUpload");
+const uploadFile = upload.single("image");
+
+const uploadImage = (req, res) => {
+  uploadFile(req, res, (err) => {
+    if (err) {
+      return res.json({
```

Comment:
May want to set status of response to 500 here (or 400 if this would instead be caused by client mistake)

Reply:
yup, 500 would be better.